src/SWP/Bundle/ElasticSearchBundle/Controller/Api/PackageSearchController.php

```diff
@@ -42,7 +42,9 @@ class PackageSearchController extends Controller
      *         {"name"="authors", "dataType"="string", "pattern"="John Doe | John Doe, Matt Smith"},
      *         {"name"="term", "dataType"="string", "pattern"="search phrase"},
      *         {"name"="sorting", "dataType"="string", "pattern"="-id|id"},
-     *         {"name"="source", "dataType"="string"}
+     *         {"name"="source", "dataType"="string"},
+     *         {"name"="per_page", "dataType"="integer"},
```

Comment:
yes, I can change it it doesnt matter really imo, I will add description for each filter in the doc yet